### PR TITLE
detailed explanation about what .m scripts and new files.

### PR DESCRIPTION
**Pull request by mselimata**
new edits to enable automatic segmentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,9 @@ For the scripts in this repo, to load the data all the relative paths aligned.
 
 First thing before the running the pipelines, add SPM to your path in MATLAB, because the scripts are calling SPM.
 
-At all the scripts there  section defining the data root. The parameter should be adjusted accordingly including true name of your data.  Here is an example setting showing the MoAEpilot folder under  ```/data/MoAEpilot``` the corresponding line in your script should look like ```root = fullfile(home, 'spmbasics', '/data/MoAEpilot')```. 
+At all the scripts there  section defining the data root which requires that ```/spmbasics``` folder to be under ```/home/username/``` directory.
```
Comment (cmaumet) [inline README.md:41]: Hi @mselimata! The issue with `/home/username` is that it's plateform specific (e.g. on MacOS would be `/Users/login`) so I'd recommend saying `to be under your home directory` instead to be cross-plateform compatible.

**Comment by cmaumet** [comment]
(Note to update this PR, you can simply push more commits to the target branch `main`)

**Comment by mselimata** [comment]
I need to figure out if there is a stochasticity between GLM estimations,  of SPM block design first level analysis. If I am not mistaken,  GUI outputs woth same settings have also difference for the files containing parameter estimates and Tstatistics.

File: README.md (hunk 1 above)
Comment (mselimata) [reply README.md]: edited accordingly, thank you.

**Pull request merged**